modules/ossm-jaeger-overview.adoc

```diff
@@ -0,0 +1,25 @@
+////
+This CONCEPT module included in the following assemblies:
+-ossm-jaeger.adoc
+////
+
+[id="ossm-jaeger-overview_{context}"]
+= Jaeger overview
+
+
+Jaeger lets service owners instrument their services to get insights into what their architecture is doing.
+Jaeger is an open source distributed tracing platform that you can use for monitoring, profiling, and troubleshooting modern, cloud-native, microservices-based applications.
```

Comment:
I'm not sure I would use "profiling" here. While I believe that this is technically correct, `profiling`, to me, implies in making a connection between the memory/CPU usage for a given method call, which isn't something that Jaeger provides natively.

Reply:
Maybe if the phrase was "distributed profiling" or something like that - focused more on latency of calls between services?

Reply:
"network profiling" then? How about something like:

> Jaeger is an open source distributed tracing platform that you can use to visualize, understand, and troubleshoot the interaction between components in a modern, cloud-native, microservices-based architecture.

Reply:
I like the term "network profiling". We need something to indicate profiling - possibly "to visualize, analyze and troubleshoot ...".

Reply:
Looks good - just a double space between "in  modern".